docs/configuration.md

```diff
@@ -211,4 +211,55 @@ telemetries: [
         }
     ],
 ]
-```
\ No newline at end of file
+```
+
+## Time to Interactive (TTI)
+
```

Comment:
issue: This section isn't structurally consistent with the other sections. Specifically, sections in this doc follow the following format:
1. (Optional) A short description of the configuration option, or plugin.
2. A table which lists configuration options.
3. A usage example.